src/test/firestore/decodeFirestoreValue.spec.js

```diff
@@ -0,0 +1,188 @@
+"use strict";
```

Comment:
This can be written in TS as well! There's a couple examples in this repo which may help